iphone/Classes/TiUIiOSProxy.m

```diff
@@ -192,6 +192,7 @@ - (NSNumber *)ROW_ACTION_STYLE_NORMAL
 
 #ifdef USE_TI_UIPICKER
 
+#if __IPHONE_13_4
```

Comment:
I'm not sure of the typical guards used on the iOS codebase. Looks like we tend to use `__IPHONE_OS_VERSION_MAX_ALLOWED`? Also, I think e may want the guards to be placed inside each method so if it's not defined we return the -1 value that we return in the runtime guard case where iOS < 13.4

Reply:
I tried to see if there was any form of existing `IS_SDK_IOS_XYZ` and the only similar thing I found was this in TiUIWebViewProxy https://github.com/appcelerator/titanium_mobile/blob/840b0d279f79248d1511fc518fa28fda9573be73/iphone/Classes/TiUIWebViewProxy.m#L619

I wasn't sure which form is right, a new `IS_SDK_IOS_134` or this one that seems to be built in (or the difference of them).

Reply:
@ewanharris  1. __IPHONE_OS_VERSION_MAX_ALLOWED is preferred way. 
    2. Positioning is fine.
    3. Need to guard in function https://github.com/appcelerator/titanium_mobile/blob/00f4b58d03678d1a266fa9a75713f938556f105a/iphone/Classes/TiUIPicker.m#L182 as well.

Reply:
Updated as requested!